assets/javascripts/discourse/initializers/post-voting-icon.js

```diff
@@ -9,8 +12,36 @@ export default {
       return;
     }
 
-    withPluginApi("1.2.0", (api) => {
-      api.replaceIcon("notification.question_answer_user_commented", "comment");
+    withPluginApi("1.18.0", (api) => {
+      if (api.registerNotificationTypeRenderer) {
```

Comment:
This function has existed for a long time now, so I don't think there's any need for the conditional